### ISSUE TITLE
Refresh not listing the flows based on Draft or Activated Status

### ISSUE THREAD

**Issue by mustaqueali** [DynamicsNinja/FlowExecutionHistory#35]
I turned on a few flows and clicked refresh, but still the flows are listed selecting Draft mode and the list not getting refreshed.  Closing the tab and opening it works fine.

**Comment by DynamicsNinja** [comment]
Yes, you are right, it fetches the data from the cache. I will implement the feature that you will need to press the button to fetch runs in general and after that, I will fix the implementation of refresh too.
